project/app/views.py

```diff
@@ -27,19 +27,14 @@ def get_queryset(self):
 
 class edit_group(UpdateView):
     model = Group
-    context_object_name = 'group'
     form_class = EditGroupForm
     template_name = 'pages/modals/editGroupDialog.html'
 
     def dispatch(self, *args, **kwargs):
         self.course_code = kwargs['pk']
-        logger = logging.getLogger()
-        logger.info('heellp')
```

Comment:
Hyvä että poistit, mutta tosiaan näitä ei kannattaisi alun perinkään commitoida niin ei sitten tarvitse tehdä erikseen committeja niiden poistamiselle 🙂